src/main/java/com/google/codeu/data/Country.java

```diff
@@ -0,0 +1,31 @@
+package com.google.codeu.data;
+
+public class Country {
```

Comment:
So, some countries are super long "The democratic republic of the Congo". I forsee that possibly causing some issues down the eline, so for now I have everything using the country codes. I do forsee a problem with compatibility once users start being able to add locations... and I'm not really sure how to deal with that quite yet. But right now, I have the country codes, and we can switch from one to another very easily if we choose to use the country names instead (if we address the spacing issues) - I guess that would be as simple as replacing spaces with nothing or underscores... I don't know. 

I partially just wanted to push this code out so that others can build on top of it, so I'm somewhat deferring these decisions until we get a little farther, because for our MVP, it doesn't matter. If it comes back to bite us on level 2, it should hopefully be a quick fix